explainability/core/counterfactuals/pom.xml

```diff
@@ -0,0 +1,66 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <artifactId>explainability-counterfactuals</artifactId>
+    <version>1.0-SNAPSHOT</version>
+
+
+    <properties>
+        <version.org.optaplanner>8.0.0-SNAPSHOT</version.org.optaplanner>
+        <maven.compiler.source>1.11</maven.compiler.source>
+        <maven.compiler.target>1.11</maven.compiler.target>
+    </properties>
+
+    <dependencyManagement>
+        <dependencies>
+            <dependency>
+                <groupId>org.optaplanner</groupId>
+                <artifactId>optaplanner-core</artifactId>
+                <version>${version.org.optaplanner}</version>
+            </dependency>
+            <dependency>
+                <groupId>org.kie.kogito</groupId>
+                <artifactId>explainability-core</artifactId>
+                <version>1.0.0-SNAPSHOT</version>
+            </dependency>
```

Comment:
I don't think we publish -SNAPSHOT artifacts to public Maven repositories so you might need to add this repository
https://github.com/kiegroup/kogito-runtimes/blob/master/pom.xml#L67-L84

This should be ok also for OptaPlanner dependency

Reply:
Updated POM